### PR TITLE
feat(devops): add develop workflow

### PR DESCRIPTION
**Pull request by rafaiovlabs**

### CODE REVIEW

**Review by 0xartem — APPROVED** [review]
looks good but needs to be tested

File: .github/workflows/develop.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+name: RIF Marketplace UI deploy
+
+on:
+  push:
+
+    branches:
+      - develop
+
+jobs:
+  deploy-staging:
```
Comment (0xartem) [inline .github/workflows/develop.yml]: deploy-develop

**Review by 0xartem — CHANGES_REQUESTED** [review]

File: .github/workflows/develop.yml (hunk 1 above)
Comment (rafaiovlabs) [reply .github/workflows/develop.yml]: done

**Review by 0xartem — APPROVED** [review]
LGTM

**Review by jurajpiar — APPROVED** [review]

**Pull request merged**